development/tools/cmd/jobguard/main.go

```diff
@@ -39,63 +39,67 @@ func main() {
 	time.Sleep(cfg.InitialSleepTime)
 
 	repoName := fmt.Sprintf("%s/%s", cfg.Status.Owner, cfg.Status.Repository)
-	jobsNumber, err := getNumberOfJobs(cfg.Prow.ConfigFile, cfg.Prow.JobsDirectory, repoName)
```

Comment:
please describe briefly what to do if someone is going to introduce a new integration job. IMO good place can be `prow-jobs.md` file.